src/_lib/sass/src/_type.scss

```diff
@@ -235,156 +270,125 @@ dd {
 blockquote.pull-right {
-  padding-right: 15px;
-  padding-left: 0;
-  border-right: 5px solid $blockquote-border-color;
-  border-left: 0;
-  text-align: right;
-
-  // Account for citation
-  footer,
-  small,
-  .small {
-    &:before { content: ''; }
-    &:after {
-      content: '\00A0 \2014'; // nbsp, em dash
+    padding-right: 15px;
+    padding-left: 0;
+    border-right: 5px solid $blockquote-border-color;
+    border-left: 0;
+    text-align: right;
+
+    // Account for citation
+    footer,
+    small,
+    .small {
+        &:before {
+            content: '';
+        }
+
+        &:after {
+            content: '\00A0 \2014'; // nbsp, em dash
+        }
     }
-  }
 }
 
 // Addresses
 address {
-  margin-bottom: $line-height-computed;
-  font-style: normal;
-  line-height: $line-height-base;
+    margin-bottom: $line-height-computed;
+    font-style: normal;
+    line-height: $line-height-base;
 }
 
 // Custom interactive text styling
-.nw-link {
-  text-decoration: none;
-
-  &.nw-link-primary {
-      color: $text-color;
-
-      &:visited,
-      &.visited {
-          color: $text-color-secondary;
-      }
-  }
-
-  &.nw-link-secondary {
-      color: $text-color-secondary;
-  }
-
-  &.nw-link-tertiary {
-      color: get-color("primary", x-light);
-
-      &:hover {
-          text-decoration: underline;
-      }
-  }
-
-  &.nw-link-inline {
-      &:visited,
-      &.visited {
-          color: $text-color-secondary;
-      }
-
-      @extend .nw-link-tertiary;
-  }
-
-  &:hover,
-  &:hover:visited,
-  &:hover.visited {
-      color: $link-hover-color;
-      cursor: pointer;
-  }
-}
-
-// TEMPORARY CUSTOM nw-text CLASS
-h1, h3, h4, h5,
-.h1, .h3, .h4, .h5 {
-  &.nw-text {
-    line-height: 1.2;
-    margin: 0;
-  }
+[class*="nw-link-"] {
+    text-decoration: none;
+    &:hover,
+    &:hover:visited,
+    &:hover.visited {
+        color: $link-hover-color;
+        cursor: pointer;
+    }
 }
 
-h1.nw-text, .h1.nw-text { font-weight: normal; font-size: 36px; }
-// No h2 for now
-h3.nw-text, .h3.nw-text { font-weight: normal; font-size: 24px; }
-h4.nw-text, .h4.nw-text { font-weight: bold; font-size: 20px; }
-h5.nw-text, .h5.nw-text { font-weight: bold; font-size: 16px; }
+.nw-link-primary {
+    color: $text-color;
 
-p {
-  &.nw-text {
-    margin: 0;
-    font-size: 14px;
-    line-height: 1.4;
-  }
-  &.text-large {
-    font-size: 16px;
-  }
-  &.text-upper {
-    letter-spacing: 0.5px;
-  }
-}
-small, .small {
-  &.nw-text {
-    margin: 0;
-    font-size: 12px;
-    line-height: 1.4;
-  }
+    &:visited,
+    &.visited {
+        color: $text-color-secondary;
+    }
 }
+
+.nw-link-secondary {
+    color: $text-color-secondary;
+}
+
+.nw-link-tertiary {
+    color: get-color("primary", x-light);
+
+    &:hover {
+        text-decoration: underline;
+    }
+}
+
+.nw-link-inline {
+    &:visited,
+    &.visited {
+        color: $text-color-secondary;
+    }
+
+    @extend .nw-link-tertiary;
+}
+
 .nw-text-light {
```

Comment:
After checking I can see that removing this class does nothing. It's used in 1 place in NWA, but since we're not updating it with the SG it makes very little difference. I will remove it from SG altogether as it is redundant.